testdata/integration_tests/SCHEDULE/SCHEDULE_COMPDAT1

```diff
@@ -55,7 +55,7 @@ COMPDAT
 COMPDAT 
 -- WELL        I    J    K1  K2            Sat.        CF       DIAM        KH SKIN ND        DIR   Ro 
      'W_1'     3    7    1    3      'OPEN'  1*     32.948      0.311   3047.839  2*         'X'     22.100 /
-     'W_1'     3    7    2    2      'OPEN'  1*     23487.0     0.311   4332.346  2*         'X'     22.123 /
+     'W_1'     3    7    2    2      'OPEN'  2      23487.0     0.311   4332.346  2*         'X'     22.123 /
```

Comment:
Add a nont-trivial `SATNUM` keyword to this deck - so tat you can check the default lookup code.